steps/containers_kube.py

```diff
@@ -2,13 +2,19 @@
 
 from behave import *
 
-@when('image "{image}" is pulled on "{host}"')
-def step_impl(context, image, host):
+@when('docker pull "{image}"')
```

Comment:
I'm leaning towards not providing the "host" option for every step. Can we rely on "target_host" for this?